### PR TITLE
[11.0][MIG] add product_template_tags

### PR DESCRIPTION
**Pull request by foutoucour**
Migration of the module to 11.0

### CODE REVIEW

File: product_template_tags/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```
Comment (Trivedi-Vacha-SerpentCS) [inline product_template_tags/models/product_template.py]: It's not needed anymore in Python 3. You can check https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

Comment (foutoucour) [reply product_template_tags/models/product_template.py]: @Trivedi-Vacha-SerpentCS fixed

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

File: product_template_tags/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Template Tags',
+    'summary': """
+        This addon allow to add tags on products""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA), Numigi',
+    'website': 'https://www.acsone.eu',
```
Comment (bodedra) [inline product_template_tags/__manifest__.py:11]: shouldn't be https://github.com/OCA/product-attribute ?

File: product_template_tags/security/product_template_rule.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bodedra) [inline product_template_tags/security/product_template_rule.xml]: Remove `encoding="utf-8"`

File: product_template_tags/views/product_template.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bodedra) [inline product_template_tags/views/product_template.xml]: Same Remove `encoding="utf-8"`

**Review by ddufresne — APPROVED** [review]

File: product_template_tags/views/product_template.xml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+            <field name="categ_id" position="after">
+                <field name="tag_ids"/>
+            </field>
+
+        </field>
+    </record>
+
+</odoo>
+
```
Comment (bodedra) [inline product_template_tags/views/product_template.xml:54]: Remove unwanted all blank spaces.

File: product_template_tags/views/product_template_tag.xml [hunk 6]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bodedra) [inline product_template_tags/views/product_template_tag.xml]: Same Remove `encoding="utf-8"`

File: product_template_tags/views/product_template_tag.xml [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_tag_form_view">
+        <field name="name">product.template.tag.form (in product_template_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
```
Comment (bodedra) [inline product_template_tags/views/product_template_tag.xml:12]: Remove `<header/>` if no issue of it.

**Review by bodedra — CHANGES_REQUESTED** [review]
Runbot is :red_circle:

**Review by max3903 — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@max3903 you have merged this having runbot with warnings, which means a red status. Please fix it. Also a module with plurals is against guidelines.

**Pull request merged**